### DISCUSSION TITLE
[docs] operating system support

### DISCUSSION THREAD

**Discussion by antelle** [tauri-apps/tauri#1211] [General]
Hi!

It would be good to have a page about operating system support (especially Windows where the webview engine is the most problematic).
From what I understand, [tauri-apps/wry](https://github.com/tauri-apps/wry) doesn't work on Windows <10, however [webview/webview](https://github.com/webview/webview) based on Microsoft `WebView2` that can be installed on Windows 7.

What is expected to be supported?

**Comment by nothingismagick** [comment]
Well, for now we will absolutely support win10. Win8 would be good, but only until it is no longer patched. This is indeed my personal philosophy, and I am somewhat of a hardliner: Do not use or in any way encourage people to use operating systems that are no longer modern and are no longer being maintained by their manufacturer. Of course, I understand that there are valid reasons for shipping apps to users who cannot feasibly upgrade their systems, but if we as an industry don't put our foot down, then how will consumers ever feel like we are protecting them and their security / privacy.

But anyway, to get to your point about webview2: it is (as it currently stands) a licensing (and spyware) catastrophe - such that I am not even sure we are not even legally allowed to wrap it and ship with their evergreen bootstrapper - without splashing a huge "Microsoft requires us to inform you that it may be tracking how you use this application." This may resolve soon, once Microsoft has shipped webview2 to all modern win10's via some patch Tuesday shenanigans, but older versions of their OS will need some special TLC, and we are still weighing our options. I will not promise you that we will ever officially support OS's that are outside of their security patching lifecycle, however.

Anyway, if you are interested, I have opened up an issue regarding licensing over at the webview2 discussion channel: 

https://github.com/MicrosoftEdge/WebView2Feedback/issues/907

**Reply by antelle** [reply]
I understand your position about not supporting Windows 7 and I agree with you, actually, Microsoft WebView2 will [drop](https://docs.microsoft.com/en-us/microsoft-edge/webview2/#supported-platforms) Windows 7 support soon. They, however, mention it in the readme of the library. That was the point of my post, please mention it somewhere on the website (perhaps "Getting Started" -> "Technical Details" can be a good place?), so that current status is visible.

Something like: https://www.electronjs.org/docs/tutorial/support#supported-platforms

The page should include: 
- OS version;
- supported architectures;
- important limitations.

**Reply by nothingismagick** [reply]
Sure, but my point is that we are still in Alpha phase, and don't really support anything yet. :) But this will be something we can do once we have a Beta label under our belts. Until then, its too soon IMHO - especially because we don't know exactly what the limitations are.

**Reply by antelle** [reply]
Makes sense, thank you for your work! 🙏 The project looks awesome so far.
